copy.go

```diff
@@ -47,7 +49,17 @@ func copy(src, dest string, info os.FileInfo, opt Options) error {
 	if skip {
 		return nil
 	}
-	return switchboard(src, dest, info, opt)
+
+	err = switchboard(src, dest, info, opt)
+
+	if opt.PreserveTimes {
+		mtime := info.ModTime()
+		stat := info.Sys().(*syscall.Stat_t)
+		atime := time.Unix(int64(stat.Atim.Sec), int64(stat.Atim.Nsec))
+		os.Chtimes(dest, atime, mtime)
```

Comment:
Should we handle error from `os.Chtimes`, even if it's less likely to happen?

Reply:
I am not sure, how would you handle it?